blocks/test/full-content.js

```diff
@@ -124,10 +123,7 @@ describe( 'full post content fixture', () => {
 			const parserOutputExpected = JSON.parse( parserOutputExpectedString );
 			expect(
 				parserOutputActual
-			).to.eql(
-				parserOutputExpected,
-				format( 'File \'%s.parsed.json\' does not match expected value', f )
-			);
+			).toEqual( parserOutputExpected );
```

Comment:
A JSON parse error is not a realistic failure mode here.  When these tests fail, we have a valid JSON object but a mismatch in its content.  Or a mismatch in the serialized string content.  It's difficult to tell the difference between these two, so we need to keep some code that prints the filename.

I see that this is done in the latest version of the PR. :+1: